app/resources/results.html.template

```diff
@@ -114,7 +114,7 @@
       {% if results_based_on_input %}
         <div class='resultsTitle'>
           {% blocktrans %}
-            Search Results for: {{query_name}} {{query_location}}
+            Search Results for: {{query}}
```

Comment:
If I understand correctly, it now outputs "Search result for: Hiroshi" instead of "Search result for: Hiroshi Tokyo" when I search for Hiroshi in Tokyo. Is that correct? That's not what we want. We still want to see "Search result for: Hiroshi Tokyo", but we do want to use the message template "Search result for: %(query)s".

I believe the best way to do that is to concatenate query_name and query_location in Python code and assign it to template parameter "query" (which is arguments of self.render).

Also, maybe you shouldn't rename query_name param to query because it may be confusing with the "query" template parameter mentioned above.

Same for other changes in this file